maas/client/viscera/machines.py

```diff
@@ -67,11 +67,17 @@ class MachinesType(ObjectType):
         else:
             return cls._object(data)
 
-    async def get_power_parameters(
-            cls, *, system_ids: typing.Sequence[str]=[]):
+    async def get_power_parameters_for(
+            cls, *, system_ids: typing.Sequence[str]):
         """
+        Get a list of power parameters for specified systems.
+        *WARNING*: This method is considered 'alpha' and may be modified
+        in future.
+
         :param system_ids: The system IDs to get power parameters for
         """
+        if system_ids == []:
+            return {}
```

Comment:
If `system_ids` were an empty tuple this branch would not be taken. Use `if len(system_ids) == 0` as the condition.